pkg/routes/route.go

```diff
@@ -66,6 +67,18 @@ func (r *Route) Consume() {
 		if err := r.pub.Publish(msg.Subject, topic, payload); err != nil {
 			r.logger.Error(fmt.Sprintf("Failed to publish on route %s: %s", r.MqttTopic, err))
 		}
-		r.logger.Debug(fmt.Sprintf("Published to: %s, payload: %s", msg.Subject, string(payload[:50])))
+		r.msgDebug(msg.Subject, payload)
 	}
 }
+
+func (r *Route) msgDebug(sub string, payload []byte) {
+	p := ""
+	l := sliceLen
+	if len(payload) > 0 {
```

Comment:
`l` is used only in `if` check, so embed it here.

Reply:
I meant:
```go
if l := sliceLen; len(payload) > 0 {
...
}
```

But from my opinion you are trying to accomplish `math.Max(sliceLen, len(payload))`, So I would make it like this:

```go
if l := math.Max(sliceLen, len(payload)); len(payload) > 0 {
   p = string(payload[:l])
}
```